charts/oes/values.yaml

```diff
@@ -281,6 +281,22 @@ sapor:
       spinAdminUsername: admin
       spinAdminPassword: admin
 
+      ## LDAP
+      ldap:
+        ldapEnabled: true
+        ldapUsername: admin
+        ldapPassword: opsmx@123
+        ldapAdminLoginEnabled: false
+        ldapAdminUsername:
+        ldapAdminPassword:
+
+      ## X509
+      x509:
+        enabled: false
+        client:
+          file: /opt/opsmx/client.p12
```

Comment:
How is this certificate mounted on a sapor pod? We should have a config map that will be mounted onto the sapor pod if x509 is enabled.

Reply:
Created new config map and enabled the mount path to sapor-x509 config map